chart/values.yaml

```diff
@@ -588,10 +588,12 @@ loki-stack:
               - __meta_kubernetes_pod_container_name
               target_label: __path__
 
+# -- Eventing component which enables or disables nats
```

Comment:
I would drop the "--" for now, not sure this would be useful for user to know nats is enabled or not, nats is an implementation detail.